app/views/course/discussion/_posts.html.slim

```diff
@@ -1,6 +1,6 @@
 - post_locals ||= {}
 - posts.each do |post|
-  = render partial: post_partial, locals: post_locals.reverse_merge(post: post.first)
+  = render partial: post_partial, object: post.first, locals: post_locals
```

Comment:
old way works, but doesn't always make sense.

if my partial is called `_discussion_post`, using `object` will expose the post as `discussion_post`, if I use local it will always be `post`